platforms/desktop/windows.md

```diff
@@ -1 +1,50 @@
-# Langley Foxall Windows Apps
\ No newline at end of file
+# Langley Foxall Windows Apps
+
+Whilst our windows application development is slightly more rare than the other platforms, we still have a set standard 
+of development, built by our in-house specialists.
+
+### Universal Windows Platform (UWP)
+
+The Universal Windows Platform is our windows platform of choice, allowing you to develop and build once for Desktop, 
+Mobile, Xbox, Hololens and more.
+
+#### Things you should know
+
+- The UWP Platform is only compatible on Windows 10, with Windows 8 having a similar framework called Windows RT.
+- The UWP & WinRT platforms use .NET Core - as opposed to the full .NET Framework. For Windows 10 this has recently
+extended to .NET Core 2.0, which added a lot of extra API's.
+
+#### Design Patterns / Architecture
```

Comment:
Might be worth mentioning views being built using XAML and which languages can be used (C++, C#, VB.NET).

Reply:
@NilesB Sorted - I've specified the preference as we have no active applications in C++ or VB.NET, and we certainly wouldn't start a new project in VB or C++, unless it was required for reasons I can't think of for LOB apps.